src/chapters/functions/composing-functions.rst

```diff
@@ -161,7 +135,7 @@ This function is still short, which is good. However, it does two separate
 jobs---it reverses a string *and* decides if that string is a palindrome.
 
 Making a palindrome checker with one function vs. two might not seem like a big
-deal, but what if we need to reverse a string for some other reason? We cannot
+deal now. But what if we need to reverse a string for some other reason? We cannot
```

Comment:
I know _technically_ it's OK to start a sentence with "But", but Mrs. Smith (my old English teacher) is screaming in my head right now.  Can we re-work this?

Reply:
We have plenty of sentences in the book that begin with conjunctions - this is just another one!

Reply:
Ha!  This made me check.  3 total for "And", "Or", and "But" combined.  I guess we'll add a fourth, but it's killing me...